ros/angel_system_nodes/angel_system_nodes/spatial_mapper.py

```diff
@@ -79,14 +80,40 @@ def __init__(self):
 
         self.poses = []
 
-        # TODO: get this from a ROS message?
-        self.image_height = 1080
-        self.image_width = 1920
+        # default image dimensions used if we fail to get the current image size
```

Comment:
I might think that we *don't* want defaults defined here because if we cannot get the image size then there may be issues with the system abound like there are no images being published yet. I imagine the current setup is not "succeeding" every time due to race conditions, so we are becoming half-reliant that these defaults are correct.